benchflow-dsl/src/main/scala/cloud/benchflow/dsl/explorationspace/ExplorationSpaceGenerator.scala

```diff
@@ -194,4 +201,75 @@ object ExplorationSpaceGenerator {
     explorationSpaceSizeOption
 
   }
+
+  /**
```

Comment:
Here is an issue for reference https://github.com/benchflow/benchflow/issues/386